framework/assets/yii.validation.js

```diff
@@ -190,25 +179,26 @@ yii.validation = (function ($) {
                 return;
             }
 
-            var valid = true;
-
-
-            var regexp = /^((?:"?([^"]*)"?\s)?)(?:\s+)?(?:(<?)((.+)@([^>]+))(>?))$/,
+            var valid = true,
+                regexp = /^((?:"?([^"]*)"?\s)?)(?:\s+)?(?:(<?)((.+)@([^>]+))(>?))$/,
                 matches = regexp.exec(value);
 
             if (matches === null) {
-                valid = false
+                valid = false;
             } else {
+                var localPart = matches[5],
```

Comment:
It's better to use separate named variables for local-part and domain because they are used more than once.